components/claim/components/email_to_oauth.jsx

```diff
@@ -74,15 +59,20 @@ export default class EmailToOAuth extends React.PureComponent {
                 }
             },
             (err) => {
-                this.setState({error: err.message, showMfa: false});
+                if (!this.state.showMfa && err.server_error_id === 'mfa.validate_token.authenticate.app_error') {
+                    this.setState({showMfa: true});
+                } else {
+                    this.setState({error: err.message, showMfa: false});
+                }
             }
         );
     }
 
     render() {
         var error = null;
         if (this.state.error) {
-            error = <div className='form-group has-error'><label className='control-label'>{this.state.error}</label></div>;
+            error =
```

Comment:
If we're breaking this into two lines, we'd generally surround the JSX with brackets. I think that might be enforced by the style checker.

Reply:
Yup, that was an auto-format mistake from the IDE. I'll change that back.